lightningd/peer_control.c

```diff
@@ -1856,3 +1851,13 @@ static const struct json_command fund_channel_command = {
 	"Returns once channel established"
 };
 AUTODATA(json_command, &fund_channel_command);
+
+const char *peer_state_name(enum peer_state state)
```

Comment:
We are not assigning enum values manually, so these will be consecutive and we could just check that `state` is smaller than the number of elements in the enum and do an array lookup, couldn't we? I'm fine with keeping it this way, just curious about the loop :wink: